src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure implements GetFigureArea {
+    private double firstLeg;
+    private double secondLeg;
+
+    @Override
+    public String toString() {

```

Comment:
According to the task description, you should not override the `toString()` method to 'draw' the figure. Instead, create a separate method to print out all information about the figure using `System.out.println()`.